src/content/docs/style-guide/quick-reference/clamshells.mdx

```diff
@@ -120,5 +120,6 @@ Here's an example of the clamshell source:
 To open or close a clamshell:
 
 * Click the open buttons or **Show/Hide All**.
-* Arrive at an individual clamshell via its URL. For example, go directly to [Clamshell 1 in the example above](/docs/new-relic-only/basic-style-guide/writing-guidelines/clamshells#clamshell-1).
+* Arrive at an individual clamshell via an anchor ID. For example, go directly to [Clamshell 1 in the example above](/docs/new-relic-only/basic-style-guide/writing-guidelines/clamshells#clamshell-1).
```

Comment:
[nit] We've renamed "clamshell" to "collapser". Should we use that language in here?

Reply:
Yup, I've updated that, thanks!